compiler/control/OMROptions.cpp

```diff
@@ -460,6 +460,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
 #ifdef J9_PROJECT_SPECIFIC
    {"disableOSRGuardRemoval",             "O\tdisable OSR guard removal",                      TR::Options::disableOptimization, osrGuardRemoval, 0, "P"},
 #endif
+   {"disableOSRLiveRangeAnalysis",        "O\tdisable live range analysis for on-stack replacement", SET_OPTION_BIT(TR_DisableOSRLiveRangeAnalysis), "F"},
```

Comment:
I take it that a more normal disable of an opt pass doesn't work because of ilgen or something?

Reply:
OSRLiveRangeAnalysis is required under certain modes for OSR transitions to be correct. It is left as an option to allow for these cases to be detected.

Reply:
Ok that makes sense - thanks for the clarification.